_posts/2013-04-08-generators-and-blueprints.md

```diff
@@ -85,38 +83,28 @@ installing
   create tests/unit/pods/foo/route-test.js
 {% endhighlight %}
 
-The built-in blueprints that support pods structure are:
-
- - adapter
- - component
- - controller
- - model
- - route
- - resource
- - serializer
- - service
- - template
- - transform
- - view
+To see which blueprints support the `--pod` option, you can use the help
+command. For example, `ember help generate component` will give you the
+list of options of the component blueprint, one of them being `--pod`.
 
 Blueprints that don't support pods structure will simply ignore the `--pod`
 option and use the default structure.
 
 If you would like to use the pods structure as the default for your project,
-you can set `usePods` in your `.ember-cli` config file to `true` (setting was
-previously named `usePodsByDefault`). To generate or destroy a blueprint in the
-classic
-type structure while `usePods` is `true`, use the `--classic` flag.
+you can set `usePods` in `.ember-cli`: 
 
-With the `usePods` set to `true`.
 {% highlight javascript %}
 // .ember-cli
 {
     "usePods": true
 }
 {% endhighlight %}
 
-The following would occur when generating a route:
+To generate or destroy a blueprint in the classic while `usePods` is
```

Comment:
classic mode?

Reply:
Thanks, I tweaked this section a bit more.